src/index.js

```diff
@@ -41,7 +40,6 @@ render((
             <Route path="/contributors" component={Contributors} />
             <Route path="/contact" component={Contact} />
             <Route path="/thanks" component={Thanks} />
-            <Route path="*" component={NotFound} />
```

Comment:
Any path following /app will load `app.html` instead of `error.html` according to the routing in `app.py`, and so the routing will get sent here instead of bringing up the error page. Removing this catch-all path removes the error page for any missing route following /app. For example, try loading http://localhost:5000/app/1.

Reply:
Good catch.